spec/factories/sales_log.rb

```diff
@@ -66,7 +66,7 @@
       sex5 { "X" }
       sex6 { "X" }
       mortgage { 20_000 }
-      ecstat3 { 10 }
+      ecstat3 { 9 }
```

Comment:
I just looked in this file and it looks like a load of magic numbers to me since I don't know how this works fully.
Would be cool to have it explained briefly.

Reply:
This file is a sort of model sales log that we use in a lot of tests (there are similar factories for other objects) - the magic numbers are just values that in theory should be part of a valid sales log in combination with each other, so usually when you're adding any new fields to the database/updating validation, you'll need to add/update a field to this file too.

Reply:
aha so there is one standard example sales log that is kind of like a version we default to, and then in any tests we just override that?
so if I call `FactoryBot.build(:sales_log)` I get a rich thing with ownership scheme 2, mortgage 20_000, person 3 with economic status 9 etc
and if I call `FactoryBot.build(:sales_log, ecstat3: 10)` I get the same thing but with person 3 economic status changed?